crates/bevy_core/Cargo.toml

```diff
@@ -21,3 +21,6 @@ bevy_math = { path = "../bevy_math", version = "0.4.0" }
 bevy_reflect = { path = "../bevy_reflect", version = "0.4.0", features = ["bevy"] }
 bevy_tasks = { path = "../bevy_tasks", version = "0.4.0" }
 bevy_utils = { path = "../bevy_utils", version = "0.4.0" }
+
+# other
+ahash = "0.6.2"
```

Comment:
`bevy_utils` re-exports `ahash::AHasher` as `bevy_utils::AHasher`, so we can use it from there instead.

Reply:
Oh nice!